Drush/EntityScaffolder/d7_1/templates/preprocess/es_helper/es_helper.module

```diff
@@ -325,3 +325,22 @@ function es_helper_preprocess_brightcove_field_embed(array &$vars) {
     $vars['video_name'] = $video_id . ':' . $video_name;
     $vars['id'] = es_helper_get_unique_video_id($video_id);
 }
+
+/**
+ * Helper function to list all file permissions for debugging purposes.
+ */
+function es_helper_show_file_permissions($dir) {
+  $files = scandir($dir);
+  foreach($files as $file) {
+    $full = $dir . '/' . $file;
+    if ($file[0] === '.') {
+      continue;
+    }
+    $perms = substr(sprintf('%o', fileperms($full)), -4);
+    print($perms . ' ' . $full. "\n");
```

Comment:
@stmh 
is this supposed to be used as drush command only? Since this uses `print` instead of `drupal_set_message`, 

So how about we either modify this to make it a drush command, OR, make it so that we can use it via devel/php? What do you think?